src/script/conversation/ConversationRepository.ts

```diff
@@ -1344,11 +1341,7 @@ export class ConversationRepository {
     const userIds = userEntities.map(userEntity => userEntity.qualifiedId);
 
     try {
-      const response = await this.conversation_service.postMembers(
-        conversationEntity.id,
-        userIds,
-        conversationEntity.isFederated(),
-      );
+      const response = await this.conversation_service.postMembers(conversationEntity.id, userIds);
```

Comment:
obviously not something you did in this 1500 line file, but im wondering if conversation_service should be conversationService

Reply:
Yeah, that's a bad call from me, actually. When I set up the core library in there, I aliased `core.service.conversation` to `conversationService`... Not realizing that the ConversationRepo already had a `conversation_service` (that will slowly be deprecated). I'll try to rename it in another PR, probably